src/dynamics/circle/rotation_number/basic.lean

```diff
@@ -0,0 +1,656 @@
+/-
+Copyright (c) 2020 Yury G. Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Yury G. Kudryashov
+-/
+import analysis.specific_limits
+import dynamics.fixed_points
+import order.iterate
+import algebra.iterate_hom
+
+/-!
+# Rotation number of a degree one circle self-map
```

Comment:
In the current version of the file, you are not defining the rotation number of a degree one circle map, but rather the translation number of a monotone real map satisfying `f (x+1) = f x + 1`. And the file is already quite long. I don't know what your plans are (add rotation number to the end of this file?) but it might make sense to move this to a standalone file `translation_number.lean` and change the main title.

Reply:
Done.